### DISCUSSION TITLE
Replace Sirens and Buzzers with one component, but with a menu for changing sounds.

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#7643] [Ideas and suggestions]
Replace the alarm buzzer and the siren with one sound component. This component would also let you pick the sound it plays from a menu. There would also be three pins, one for loop sound, play sound once, and change sound based on numeric input.

The goal of doing this would be to streamline the process of sound components. Modders could then just add sounds, rather than one component per sound, potentially saving on redundant components.

**Comment by gerakis** [comment]
With a complex enough wiring setup you could play music, minecraft style.
